bearer/bearer.go

```diff
@@ -52,6 +52,7 @@ func (b *Generator) NewBearer(email string, currentEpoch uint64, msPerEpoch int6
 	rec.AddFilter(eacl.HeaderFromObject, eacl.MatchStringNotEqual, object.AttributeContentType, "text/html")
 	rec.AddFilter(eacl.HeaderFromObject, eacl.MatchStringNotEqual, object.AttributeContentType, "text/htmlh")
 	rec.AddFilter(eacl.HeaderFromObject, eacl.MatchStringNotEqual, object.AttributeContentType, "")
+	rec.AddFilter(eacl.HeaderFromObject, eacl.MatchNotPresent, object.AttributeContentType, "")
```

Comment:
Looks like it should be inverted. We're conjugating things to allow here and we're allowing an object if ContentType is present (and not allowing if it's missing).

Reply:
Is it possible to invert condifition? I thought it wasn't possible, thus there are `eacl.MatchStringNotEqual` and opposite `eacl.MatchStringEqual`.
There is no `MatchPresent` option. Any hidden options?

Reply:
It can be expressed as another rule (DENY if not present), but I'm not sure about filter inversion. @carpawell?

Reply:
yes, you should deny objects that `MatchNotPresent` if you want to skip objects without `AttributeContentType`. and yes, we can think about `MatchPresent` too (currently there is no such one)